korge/src/commonMain/kotlin/korlibs/korge/view/CachedContainer.kt

```diff
@@ -36,7 +36,9 @@ open class FixedSizeCachedContainer(
 
 open class CachedContainer(
     @property:ViewProperty
-    var cache: Boolean = true
+    var cache: Boolean = true,
+    @property:ViewProperty
+    var expensiveScaling: Boolean? = null,
```

Comment:
I'm not sure about this name. Could we pass a `quality: Quality` enum, or rename it? Like `hidpiAware: Boolean?` or other names? People might ask: In which way is this expensive? What scaling mean in this context?